policy/modules/roles/sysadm.te

```diff
@@ -38,11 +42,22 @@ allow sysadm_t self:netlink_route_socket rw_netlink_socket_perms;
 # for debugging BPF programs
 allow sysadm_t self:capability2 bpf;
 
+allow sysadm_t self:netlink_generic_socket { create setopt bind write read };
+
+# for ptrace
+allow sysadm_t self:netlink_tcpdiag_socket { create write nlmsg_read read };
+
+allow sysadm_t self:capability audit_write;
+allow sysadm_t self:system status;
```

Comment:
This doesn't seem to fit. Typically, `system` userspace perms only apply to systemd system or user sessions. There is already `allow sysadm_t sysadm_systemd_t:system { disable enable reload start status stop };` in the policy, so there may be a labeling problem on your system. It may also explain some of the other perms you've added.